client/components/VideoChat.jsx

```diff
@@ -15,7 +15,7 @@ class VideoChat extends React.Component {
 
   componentDidMount() {
     const constraints = {
-      audio: true,
+      audio: false, // ZS set this to false so that annyang can access the mic
```

Comment:
TODO:
Is it possible to have video chat using audio at the same time?

I.e. would it be possible to turn off the audio whenever the mic hears a 'trigger word' like 'anyang' and then switches to listening for commands?